lib/elixir_conf_africa/users/user.ex

```diff
@@ -0,0 +1,163 @@
+defmodule ElixirConfAfrica.Users.User do
+  @moduledoc false
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "users" do
+    field :email, :string
+    field :password, :string, virtual: true, redact: true
+    field :hashed_password, :string, redact: true
+    field :confirmed_at, :naive_datetime
+    field :first_name, :string
+    field :last_name, :string
+    field :country, :string
```

Comment:
we've not discussed db strucure, but I would suggest we have an accounts db
this is too much info for a users table

Reply:
Okay , I will wait for the db structure discussion at 6 pm then restructure accordingly